srcpkgs/bitwig-studio/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'bitwig-studio'
+pkgname=bitwig-studio
+version=3.2.6
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+depends="xdg-utils xcb-util-wm libbsd"
+short_desc="DAW for music production, remixing and live performance"
+maintainer="Alessandro Saglia <ale.saglia@gmail.com>"
+license="custom:bitwig"
+homepage="bitwig.com"
+_file="bitwig-studio-${version}.deb"
+distfiles="https://downloads.bitwig.com/stable/${version}/${_file}"
+checksum=d60e288e1ff1fd4b716843b3315ea68d7cbe36e67ca92517457a682bed5b1902
+nostrip=yes
+restricted=yes
+repository=nonfree
+
+do_extract() {
+	mkdir -p ${DESTDIR}
+	ar x ${XBPS_SRCDISTDIR}/bitwig-studio-${version}/${_file}
+}
+
+do_install() {
+	tar xf data.tar.xz -C ${DESTDIR}
```

Comment:
You could just as well use `bsdtar` which is available w/o adding it to `hostmakedepends`.

Reply:
Fixed with last commit